src/views/hoc/hoc.scss

```diff
@@ -8,123 +8,117 @@ $base-bg: $ui-white;
+            margin-right: 15px;
+        }
 
-            .card-deck {
-                padding: 0 20px;
-            }
+        &.mod-studio {
+            float: left;
+        }
 
-            //6 columns
-            @media only screen and (min-width: $mobile) and (max-width: $tablet - 1) {
-                max-width: $mobile;
-            }
+        &.mod-logo {
+            margin: 20px;
+            max-width: 150px;
+            max-height: 55px;
+            vertical-align: middle;
         }
     }
 
-    .flex-row {
-        .card {
-            margin: 10px;
-            border-radius: 7px;
-            background-color: $active-gray;
-            padding: 2px;
-
-            width: 30%;
-            min-width: 200px;
-            max-width: 230px;
-
-            .card-info {
-                border-radius: 5px;
-                background-color: $base-bg;
-                width: 100%;
-                height: 100%;
-
-
-                button,
-                img {
-                    width: calc(100% - 20px);
-                }
-
-                img {
-                    margin: 10px 10px 5px 10px;
-                    border-radius: 5px;
-                }
-
-                button {
-                    margin: 0 10px 10px 10px;
-                }
-            }
-
-            //8 columns
-            @media only screen and (min-width: $tablet) and (max-width: $desktop - 1) {
-                margin: 2px;
-                min-width: 175px;
-
-                button {
-                    font-size: .7em;
-                }
-            }
-
+    .button {
+        &.card-info {
+            margin: 0 10px 10px 10px;
         }
     }
 
-    section {
+    .section {
         display: flex;
         margin: 0 auto;
         border-bottom: 1px solid $ui-border;
         padding: 50px 0;
         width: 95%;
+        text-align: center;
 
         justify-content: center;
         flex-wrap: wrap;
         align-items: center;
 
-        p {
-            margin: 10px auto 20px;
-            max-width: 600px;
-        }
-
         &:last-child {
             border-bottom: 0;
         }
 
-        &.one-up {
-            text-align: center;
+        &.title-banner-section {
+            border: 0;
+            padding: 10px 0;
+            max-width: $desktop;
+        }
+    }
 
-            .column {
-                width: 100%;
-            }
+    .section-column {
+        width: 100%;
+
+        &.mod-split {
+            min-width: 200px;
+            max-width: 40%;
+            text-align: left;
         }
+    }
 
-        &.two-up {
-            .column {
-                min-width: 200px;
-                max-width: 40%;
-                text-align: left;
+    .section-paragraph {
+        margin: 10px auto 20px;
+        max-width: 600px;
+    }
 
-                img {
-                    border-radius: 5px;
-                    width: 100%;
-                }
-            }
+    .title-banner {
+        &.wbb-bg {
```

Comment:
I left it as is since it seemed to be doing the modifier. I guess I should've added the `mod-` prefix, though. I can add that in.